### PR TITLE
Remove C-Thread-Pool

### LINKED ISSUE #21: Remove C-Thread-Pool
Some executables still use C-Thread-Pool instead of QueuePerThreadPool.

### PR DESCRIPTION
**Pull request by calccrypto**
Completely removed C-Thread-Pool from the code.

Fixes #21

### CODE REVIEW

**Comment by calccrypto** [comment]
Can the mentions of C-Thread-Pool in the licenses be removed?

File: include/utils.h [hunk 1]
```diff
@@ -174,7 +162,9 @@ typedef void(DirFunc)(void*);
 int processdirs(DirFunc dir_fn);
```
Comment (dmanno) [inline include/utils.h]: cleanup

Comment (dmanno) [reply include/utils.h]: I think you removed the need for this, right?

Comment (dmanno) [reply include/utils.h]: bfmi was not updated and still requires the old functions for C-Thread-Pool.

Comment (calccrypto) [reply include/utils.h]: 50687c1

**Pull request merged**